src/components/BallotProxyMetadata/index.js

```diff
@@ -16,11 +17,12 @@ export class BallotProxyMetadata extends React.Component {
       /*4*/ { value: '4', label: ballotStore.ProxyBallotType[4] }, // VotingToChangeProxy
       /*5*/ { value: '5', label: ballotStore.ProxyBallotType[5] }, // BallotsStorage
       /*6*/ { value: '7', label: ballotStore.ProxyBallotType[7] }, // ValidatorMetadata
-      /*7*/ { value: '8', label: ballotStore.ProxyBallotType[8] } // ProxyStorage
+      /*7*/ { value: '8', label: ballotStore.ProxyBallotType[8] }, // ProxyStorage
+      /*8*/ { value: '9', label: ballotStore.ProxyBallotType[9] } // RewardByBlock
```

Comment:
I'm not sure because we already have the enum here: https://github.com/poanetwork/poa-dapps-voting/blob/1b7ae3e94b08753bc9da4fea443690fe32cb2af1/src/stores/BallotStore.js#L22-L31

Reply:
Ok, let's leave it as is. I see, that some lines of source code dependent on this structure of `ProxyBallotType` object.